### DISCUSSION TITLE
React Native Health Discord Link

### DISCUSSION THREAD

**Discussion by mgfuller333** [agencyenterprise/react-native-health#215] [Ideas] [answered]
Here is the link to the React Native Discord.

https://discord.gg/NxUGuvKB

Let's schedule a community wide meeting to get to know each other as developers.

Let's also learn about how we have created some cool things with React Native Health.

I also would like to have a discussion on the direction that we can take React Native Health with the recent updates to
Apple Healthkit.

**Comment by mgfuller333** [comment]
https://discord.gg/NxUGuvKB - I created one!

**Comment by GGGava** [comment, marked as answer]
Thanks for the idea @mgfuller333 

The current official server is https://discord.gg/SY3m9whWqV
